content/en/cloud_cost_management/aws.md

```diff
@@ -135,20 +135,111 @@ Attach the new S3 policy to the Datadog integration role.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
 
-### Cost types
+## Cost types
 
-You can visualize your ingested data using the following cost types:
+Visualize your ingested data using out-of-the-box cost types. Cloud cost metric names contain various terms that describe how they are calculated.
 
-| Cost Type            | Description           |
+### On-demand
+**On-demand** costs are the costs of usage at the public, on-demand rate published by AWS. This excludes all savings plans, reservations, discounts, taxes, and fees.
+
+On-demand costs do not accurately represent what you pay AWS. Do not use this metric to estimate your actual costs.
+
+### Amortized and unblended costs
+**Amortized** cost metrics distribute some pre-payments (like up-front savings plan fees) across the entire discount term. This is also called _accrual basis_. As savings plans are applied to covered usage, the costs for that savings plan appear as usage charges. The unused portion appears as a fee.
+
+Not all one-time fees are amortized across the billing period. For example, the unused portion of a Reserved Instance Fee typically appears on the first of the month.
+
+**Unblended** cost metrics show all charges on the date that they were incurred. This is also called _cash basis_. These metrics match the AWS invoice exactly.
+
+### Net costs
+**Net** costs apply private discounts directly to usage. The cost of usage for a specific resource represents the effective cost after all savings are realized.
+
+Other metrics (without the `net` keyword) show private discounts as separate, negative-valued line items with no attribution tags. Rather than attributing the discounts to actual usage, they are treated as offsets to total cost.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Perhaps we should refrain of talking about keywords and suffixes. I believe our overall goal is to hide the name of the metrics (once we migrate the new searchbar to dashboards).
If we don't have a better solution now, we can revisit that later

Reply:
great point, I will try to revise this. I will leave the full metric names in the Metric Summary table for now, and we can modify that when they're no longer visible if we want.

Though, I'm guessing you will still see them in string-query edit mode?